lib/measured/util.rb

```diff
@@ -0,0 +1,24 @@
+module Measured
+  class << self
+    alias_method :old_build, :build
+
+    def build(**kwargs, &block)
+      measurable = old_build(**kwargs, &block)
+      add_extensions(measurable)
+      measurable
+    end
+
+    private
+
+    def add_extensions(measurable)
+      measurable.units_with_aliases.each do |name|
```

Comment:
How do other libraries that do things like this? Seems like a lot of core extensions just open up the class and place the methods in there, which would be equivalent to clobbering the old methods.

If we raise an error because of an existing method, it may not be between units (e.g., what if `dup` was the name of a unit?). Given the dynamic nature of these names, it's probably a good idea to raise if it already exists in any case, because changing an existing, non-`Measurable` method would be super confusing and hard to debug.